OpenRA.Mods.D2k/UtilityCommands/D2kMapImporter.cs

```diff
@@ -0,0 +1,357 @@
+			{ 591, Pair.New("REPAIRH", "Corrino") },              //// Emperor repair pad
+			{ 592, Pair.New("GUNTOWERH", "Corrino") },            //// Emperor gun turret
+			{ 593, Pair.New("HIGHTECHH", "Corrino") },            //// Emperor high tech factory
+			{ 594, Pair.New("ROCKETTOWERH", "Corrino") },         //// Emperor rocket turret
+			{ 595, Pair.New("RESEARCHH", "Corrino") },            //// Emperor Ix research center
+			{ 596, Pair.New("STARPORTC", "Corrino") },            //// Emperor starport
+			////{597, Pair.New("", "Corrino") },// Emperor sietch
+			{ 598, Pair.New("RIFLE", "Corrino") },                //// Emperor light infantry
+			{ 599, Pair.New("BAZOOKA", "Corrino") },              //// Emperor trooper
+			{ 600, Pair.New("SARDAUKAR", "Corrino") },            //// Emperor sardaukar
+			{ 601, Pair.New("FREMEN", "Corrino") },               //// Emperor fremen
+			{ 602, Pair.New("ENGINEER", "Corrino") },             //// Emperor engineer
+			{ 603, Pair.New("HARVESTER", "Corrino") },            //// Emperor harvester
+			{ 604, Pair.New("MCVH", "Corrino") },                 //// Emperor MCV
+			{ 605, Pair.New("TRIKE", "Corrino") },                //// Emperor trike
+			{ 606, Pair.New("QUAD", "Corrino") },                 //// Emperor quad
+			{ 607, Pair.New("COMBATH", "Corrino") },              //// Emperor combat tank
+			{ 608, Pair.New("MISSILETANK", "Corrino") },          //// Emperor missile tank
+			{ 609, Pair.New("SIEGETANK", "Corrino") },            //// Emperor siege tank
+			{ 610, Pair.New("CARRYALLH", "Corrino") },            //// Emperor carryall
+
+			// Fremen:
+			{ 620, Pair.New("WALLA", "Fremen") },                 //// Fremen wall
+			{ 621, Pair.New("PWRA", "Fremen") },                  //// Fremen wind trap
+			{ 622, Pair.New("CONYARDA", "Fremen") },              //// Fremen construction yard
+			{ 623, Pair.New("BARRA", "Fremen") },                 //// Fremen barracks
+			{ 624, Pair.New("REFA", "Fremen") },                  //// Fremen refinery
+			{ 625, Pair.New("RADARA", "Fremen") },                //// Fremen outpost
+			{ 627, Pair.New("LIGHTA", "Fremen") },                //// Fremen light factory
+			{ 628, Pair.New("PALACEC", "Fremen") },               //// Imperial palace under Fremen control
+			{ 629, Pair.New("SILOA", "Fremen") },                 //// Fremen silo
+			{ 630, Pair.New("HEAVYA", "Fremen") },                //// Fremen heavy factory
+			////{ 631, Pair.New("REPAIRA", "Fremen") },// Fremen repair pad
+			{ 632, Pair.New("GUNTOWERA", "Fremen") },             //// Fremen gun turret
+			{ 633, Pair.New("HIGHTECHA", "Fremen") },             //// Fremen high tech factory
+			{ 634, Pair.New("ROCKETTOWERA", "Fremen") },          //// Fremen rocket turret
+			{ 635, Pair.New("RESEARCHA", "Fremen") },             //// Fremen Ix research center
+			////{ 636, Pair.New("REPAIRA", "Fremen") },// Fremen repair pad
+			////{ 637, Pair.New("", "Fremen") },// Fremen sietch
```

Comment:
"SIETCH" should work. It is already set up. https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/rules/structures.yaml#L705